CHANGELOG.md

```diff
@@ -1,5 +1,97 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.36
+
+<!-- generated comparing v5.0.0-alpha.35..next -->
+
+_Jun 7, 2021_
+
+Big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
+
+We've introduced a new `ListItemButton` component that should prevent common mistakes when using `<ListItem button />` and help with customization and TypeScript support.
+
+`experimentalStyled` is now available without the `experimental` prefix.
+We're confident that it's API shouldn't receive any major changes until the stable release of v5.
+
+`@material-ui/icons` only ships ES modules and no longer CommonJS modules.
+This reduces the download size of the package from 1.7 MB to 1.2 MB and install size from 15.6 MB to 6.7 MB.
```

Comment:
```suggestion
- ⚒️ We've introduced a new `ListItemButton` component that should prevent common mistakes when using `<ListItem button />` and help with customization and TypeScript support.
- 👩‍🎤 `experimentalStyled` is now available without the `experimental` prefix.
We're confident that it's API shouldn't receive any major changes until the stable release of v5.
- 📦 `@material-ui/icons` only ships ES modules and no longer CommonJS modules.
This reduces the download size of the package from 1.7 MB to 1.2 MB and install size from 15.6 MB to 6.7 MB.
```

To be closer to previous releases' description

Reply:
How about we also link the PR so developers can dive if needed?

```suggestion
- ⚒️ We've introduced a new `ListItemButton` component that should prevent common mistakes when using `<ListItem button />` and help with customization and TypeScript support (#26446) @siriwatknp.
- 👩‍🎤 `experimentalStyled` is now available without the `experimental` prefix.
We're confident that its API shouldn't receive any major changes until the stable release of v5 (#26558) @mnajdova.
- 📦 `@material-ui/icons` only ships ES modules and no longer CommonJS modules.
This reduces the download size of the package from 1.7 MB to 1.2 MB and installs size from 15.6 MB to 6.7 MB (#26310) @eps1lon.
```

Reply:
I think that we should talk about #26559 too, its issue had over 30 upvotes